pkg/v1/providers/ytt/02_addons/cni/antrea/antrea_addon_data.lib.yaml

```diff
@@ -54,6 +54,10 @@ antrea:
     #@ end
     tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
     disableUdpTunnelOffload: #@ data.values.ANTREA_DISABLE_UDP_TUNNEL_OFFLOAD
+    #! Setting defaultMTU to null since antrea-agent will discover the MTU of the Node's primary interface and
+    #! also adjust MTU to accommodate for tunnel encapsulation overhead.
+    #@overlay/match missing_ok=True
+    defaultMTU: #@ str(data.values.DEFAULT_MTU)
```

Comment:
Antrea and Calico can't share this DEFAULT_MTU param because it's a integer in Antrea, not a string. The description of it can be found here: https://github.com/antrea-io/antrea/blob/main/pkg/config/agent/config.go#L77